src/compiler/crystal/syntax/parser.cr

```diff
@@ -4126,14 +4131,22 @@ module Crystal
       named_args
     end
 
-    def parse_call_arg
+    def parse_call_arg(found_double_splat = false)
       if @token.keyword?(:out)
+        if found_double_splat
+          raise "out not allowed after double splat"
```

Comment:
Could we be a little more verbose here? Like:

`out arguments are not allowed after double splats`

Reply:
Fixed!